src/ExportHistory/Exception/ExportJobClientValidationException.cs

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.DurableTask.ExportHistory;
+
+/// <summary>
+/// Exception thrown when client-side validation fails for export job operations.
+/// </summary>
+public class ExportJobClientValidationException : InvalidOperationException
+{
+    /// <summary>
+    /// Initializes a new instance of the <see cref="ExportJobClientValidationException"/> class.
+    /// </summary>
+    /// <param name="jobId">The ID of the export job that failed validation.</param>
+    /// <param name="message">The validation error message.</param>
+    public ExportJobClientValidationException(string jobId, string message)
+        : base($"Validation failed for export job '{jobId}': {message}")
+    {
+        this.JobId = jobId;
+    }
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="ExportJobClientValidationException"/> class.
+    /// </summary>
+    /// <param name="jobId">The ID of the export job that failed validation.</param>
+    /// <param name="message">The validation error message.</param>
+    /// <param name="innerException">The exception that is the cause of the current exception.</param>
+    public ExportJobClientValidationException(string jobId, string message, Exception innerException)
```

Comment:
Just for code simplification, it seems like you could just do a single constructor with an optional exception: `ExportJobClientValidationException(string JobId, string message, Exception? innerException = null)`

Unless that freaks out the base compiler, in which case, disregard this. If it does work though, please do it for all your exceptions.